frontend/src/metabase-types/api/settings.ts

```diff
@@ -201,7 +201,7 @@ interface InstanceSettings {
   "admin-email": string;
   "email-smtp-host": string | null;
   "email-smtp-port": number | null;
-  "email-smtp-security": "None" | "SSL" | "TLS" | "STARTTLS";
+  "email-smtp-security": "none" | "ssl" | "tls" | "starttls";
```

Comment:
these values actually match what the backend expects, the previous values were display values